ui/src/pages/help-center/help-center-card/HelpCenterCard.types.ts

```diff
@@ -0,0 +1,20 @@
+export interface CardButton {
+  text: string;
+  target: string;
+  url: string;
+  startIconName: string;
+}
+
+export interface CardType {
+  id: string;
+  title: string;
+  description: string;
+  buttons: Array<CardButton>;
```

Comment:
I think the square bracket declaration for arrays is more [preffered](https://graphite.dev/guides/typescript-arrays). eq.

```ts
buttons: HelpCardButton[]
```